console_logging.go

```diff
@@ -88,13 +88,24 @@ var (
 	Color = false
 )
 
+func IsValid(file *os.File) bool {
```

Comment:
Please add a comment, but also rename to something meaningful

Maybe


```suggestion
func IsValidFile(file *os.File) bool {
```

Reply:
I probably should make it `isValid` just figured it may be useful but yes if I leave it exposed, it should have a doc

Reply:
made private but also added comment